### PR TITLE
Implement AV Inputs

### PR DESCRIPTION
**Pull request by llPlayll**

### CODE REVIEW

File: Assets/Editor/Resources/WorkshopItem.asset
Comment (GoodHood1) [inline Assets/Editor/Resources/WorkshopItem.asset:1]: Why is this file changed?

File: Assets/Editor/Resources/WorkshopItem.asset.meta
Comment (GoodHood1) [inline Assets/Editor/Resources/WorkshopItem.asset.meta:1]: And this one too?

File: Assets/Editor/Resources/WorkshopItem.asset
Comment (llPlayll) [reply Assets/Editor/Resources/WorkshopItem.asset:1]: I went into the Steam Workshop tab to check where the module is built, which created that file along with the .meta one.

**Comment by Gameplushy** [comment]
Closing in favor of #13
